Wikipedia/UI-V5/WMFHomeNearbyCell.m

```diff
@@ -0,0 +1,162 @@
+
+#import "WMFHomeNearbyCell.h"
+
+#import "Wikipedia-Swift.h"
+#import "PromiseKit.h"
+
+@interface WMFHomeNearbyCell ()
+
+@property (strong, nonatomic) IBOutlet UIImageView *imageView;
```

Comment:
i mentioned this in @montehurd's PR, but I usually make my subviews `weak`. I'll flag it for discussion at this week's Code Review